generated-code-samples-for-review/language/language_sentiment_text.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by protoc-gen-go_gapic. DO NOT EDIT.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	language "cloud.google.com/go/language/apiv1"
+	languagepb "google.golang.org/genproto/googleapis/cloud/language/v1"
+)
+
+// [START language_sentiment_text]
```

Comment:
Should be above the imports. But, the imports include `flag`, which isn't used in the actual sample (only `func main`).

Reply:
@hzyi-google

I've seen us do the following with samples:

```
imports (core imports which we don't want to include in embedded snippet)
[START the_sample_code]
imports (MOAR imports ... these are the ones we want to document and show users, e.g. to include the client library)
```

But if `flag` is no longer needed at some point in the same .go file, it becomes a non-issue.

What would the follow up on this one be?